content/applications/inventory_and_mrp/maintenance/maintenance_calendar.rst

```diff
@@ -0,0 +1,378 @@
+being viewed before clicking it.
+
+At the far-right side of the page is a sidebar column, containing a minimized calendar set to
+today's date, and a drop-down :guilabel:`Technician` list, displaying all the *Technicians* with
+requests currently open. Click the :guilabel:`(panel)` icon at the top of this sidebar to open or
+close the sidebar.
+
+.. image:: maintenance_calendar/maintenance-calendar-collapsible-panel.png
+   :align: center
+   :alt: Collapsible sidebar panel icon with minimized calendar.
+
+.. note::
+   The :guilabel:`Technician` list will only display if *Technicians* are assigned to open requests,
+   and individual *Technicians* will only be listed if they are listed as :guilabel:`Responsible` on
+   at least **one** maintenance request form.
+
+Kanban view
+~~~~~~~~~~~
+
+With the :guilabel:`Kanban` view, all open maintenance requests are displayed in Kanban-style
+columns, in their respective stages of the maintenance process.
+
+Each maintenance request appears on its own task card, and each task card can be dragged and dropped
+to a different stage of the Kanban pipeline.
+
+Each column has a name (i.e. :guilabel:`In Progress`). Hovering at the top of a column reveals a
+:guilabel:`⚙️ (gear)` icon. Clicking the :guilabel:`⚙️ (gear)` icon reveals a list of options for
+that column: :guilabel:`Fold`, :guilabel:`Edit`, :guilabel:`Automations`, and :guilabel:`Delete`.
+
+.. image:: maintenance_calendar/maintenance-calendar-kanban-column.png
+   :align: center
+   :alt: Column options for stage in Kanban view.
+
+Clicking :guilabel:`Fold` folds the column to hide its contents.
+
+Clicking :guilabel:`Edit` opens an :guilabel:`Edit:` pop-up window, with the corresponding *Stage*
+name, wherein the column's details can be edited. The following are the column options that can be
+edited:
+
+.. image:: maintenance_calendar/maintenance-calendar-edit-stage-popup.png
+   :align: center
+   :alt: Edit In Progress pop-up window.
+
+- :guilabel:`Name`: the name of the *Stage* in the Kanban pipeline.
+- :guilabel:`Folded in Maintenance Pipe`: when checked, this *Stage's* column will be folded by
+  default in the :guilabel:`Kanban` view type.
+- :guilabel:`Request Confirmed`: when this box is not ticked, and the maintenance request type is
+  set to *Work Center*, no leave is created for the respective work center when a maintenance
+  request is created. If the box is ticked, the work center is automatically blocked for the listed
+  duration, either at the specified date, or as soon as possible, if the work center is unavailable.
+- :guilabel:`Sequence`: the order in the maintenance process in which this *Stage* appears.
+- :guilabel:`Request Done`: if ticked, this box indicates this *Stage* is the final step in the
+  maintenance process. Requests moved to this *Stage* will close.
+
+Once ready, click :guilabel:`Save & Close`. If no changes have been made, click :guilabel:`Cancel`.
```

Comment:
there is no 'Cancel' -- there is a discard button, though. is that what you're referring to? Also, if no changes were made, couldn't they simply click out ('X' out of the pop-up) to get the same result? A little clarification here would be beneficial

Reply:
Correct. Suggestion implemented ✅